### DISCUSSION TITLE
Apollo Save Tool v1.7.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps3#84] [General]
### Added

* Added local Web Server (download saves as .Zip)
* Save-game selection (press `SELECT` to tag items)
  * Copy/Resign selected saves (Bulk Management)
* Improved listings (items count)
* New custom checksum support
  * LEGO Marvel Avengers
  * LEGO Star Wars: The Force Awakens

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps3/releases/tag/v1.7.0'>Apollo Save Tool v1.7.0</a>.</em>

**Comment by Horazion720** [comment]
Dude

**Reply by Masamune3210** [reply]
Please don't start posting on random things in the repo, just because you didnt get the response to your frankly malicious issue report that you wanted.....
